packages/components/src/Tooltip/_docs/Tooltip.stories.tsx

```diff
@@ -12,6 +21,13 @@ const meta = {
   args: {
     text: "Example tooltip text.",
   },
+  decorators: [
+    Story => (
+      <div className="flex mt-[50px] justify-center gap-12">
+        <Story />
+      </div>
+    ),
+  ],
```

Comment:
this isn't the right way to center story. Doing it messes up with screenshots, responsiveness, some docs addons and other things. You can use `layout: "centered"`  instead. https://storybook.js.org/docs/configure/story-layout#component-layout

Reply:
ah nice, was just updating the original one that used inline styles so this should reduce the bloat further :) Cheers for the tip 👍